projects/azure/RELEASES.md

```diff
@@ -11,8 +11,8 @@ All Azure Red Hat OpenShift (ARO) versions:
 |release-v4|v4.3      |deprecated |
 |release-v4|v4.4      |deprecated |
 |release-v5|v5.1      |deprecated |
-|release-v5|v5.2      |live       |
-|master    |v6.0      |development|
+|release-v5|v5.2      |deprecated |
+|master    |v6.0      |live       |
 
 # Testing matrix
```

Comment:
I think, this is outdated. Should we remove this section?

Reply:
😬  That makes sense. 🤦‍♂

When I saw the job failing the _other_ PR it caused concern.  

As far as the section... I'm not quite sure but I'm open to removing it.

Reply:
I think, we should either update the section with up-to-date information or remove it. I personally do not see much value in maintaining this piece of documentation.

P.S.: I think, I'm guilty for not doing it when I was removing v4.

Reply:
v7.0 is released as well but not live.